build/init/fetch-linux-headers.sh

```diff
@@ -32,7 +36,9 @@ fetch_cos_linux_sources()
 
 fetch_generic_linux_sources()
 {
-  kernel_version="$(echo "${KERNEL_VERSION}" | awk -vFS=+ '{ print $1 }')"
+  # 4.19.76-linuxkit -> 4.19.76
+  # 4.14.154-128.181.amzn2.x86_64 -> 4.14.154
+  kernel_version="$(echo "${KERNEL_VERSION}" | awk -vFS=- '{ print $1 }')"
```

Comment:
on GCP the + is used, can we have it field separate on - or +?

Reply:
Would you be able to give some examples of GCP values for KERNEL_VERSION and what they should be translated to? I can see about making this bit work for those as well.

Reply:
`4.19.76+` I think the + means "extra commits from vendor" in this case, google?

Reply:
for this we just want `4.19.76` i think, literally just chopping off the `+`.

Reply:
Thanks, updated to support both!